ui/src/buckets/components/BucketsTab.tsx

```diff
@@ -191,17 +217,25 @@ class BucketsTab extends PureComponent<Props, State> {
   }
 }
 
-const mstp = ({buckets, orgs}: AppState): StateProps => {
-  return {
-    buckets: buckets.list,
-    org: orgs.org,
-  }
-}
+const mstp = ({
+  buckets,
+  orgs,
+  cloud: {
+    limits: {
+      buckets: {limitStatus},
+    },
+  },
```

Comment:
This is a bunch of deconstruction with no protection should one of these keys not exist.